linker/linker_test.go

```diff
@@ -39,20 +39,10 @@ import (
 	"github.com/bufbuild/protocompile/reporter"
 )
 
-var (
-	protocPath = "../internal/testdata/protoc/22.0/bin/protoc"
-	nullDevice = "/dev/null"
+const (
+	nullDevice = os.DevNull
```

Comment:
I would just remove this entirely and update the one reference below (~line 2251) to directly use `os.DevNull`.